Kconfig.thingset

```
@@ -43,6 +50,13 @@ config THINGSET_OBJECT_LOOKUP_BUCKETS
 
 endif
 
+config THINGSET_SEND_0DP_FLOATS_AS_INTEGERS
```

When I first looked at this I read the 0 as the letter O. Can we replace this with something like this to be more unambiguous?

```suggestion
config THINGSET_ENCODE_ZERO_DECIMAL_FLOATS_AS_INTEGERS
```